README.md

```diff
@@ -1,340 +1,535 @@
+        - [Launch with YAML config file](#launch-with-yaml-config-file)
+    - [Deposit and register Validator](#deposit-and-register-validator)
+    - [Troubleshooting](#troubleshooting)
+      - [dial tcp timeout](#dial-tcp-timeout)
+      - [invalid URI for request](#invalid-uri-for-request)
+      - [connection refused](#connection-refused)
+  - [Operator Quick start](#operator-quick-start)
+    - [Pre requisites](#pre-requisites)
+    - [Start a DKG-operator](#start-a-dkg-operator)
+      - [Launch with Docker and YAML file](#launch-with-docker-and-yaml-file-1)
+      - [Build from source](#build-from-source-1)
+        - [Build](#build-1)
+        - [Launch with command line parameters](#launch-with-command-line-parameters-1)
+        - [Launch with YAML config file](#launch-with-yaml-config-file-1)
+    - [Update Operator metadata](#update-operator-metadata)
+  - [Flow Description:](#flow-description)
+    - [Note on DKG instance management](#note-on-dkg-instance-management)
+  - [Security notes](#security-notes)
+
+## Goal and Introduction
+
+> 🚫 This tool was not audited.
+> 
+> When using distributed key generation you understand all the risks involved with experimental cryptography. 
+
+### DKG
+
+Distributed Key Generation is a cryptographic process that aims to solve the problem of coordinating N parties to cryptographically sign and verify signatures without relying on Trusted Third Parties. The process is demonstrated to be successful in computing a key pair in the presence of a number T attackers in a decentralized network. To do so, this algorithm generates a public key, and a secret key of which no single party knows, but has some share of. The involvement of many parties requires Distributed key generation to ensure secrecy in the presence of malicious contributions to the key calculation.
+For more information about DKG in general, [please visit this page](https://en.wikipedia.org/wiki/Distributed_key_generation).
+
+### DKG tool by SSV
+
+The SSV team built this tool leveraging [drand](https://drand.love/)'s DKG protocol implementation ([please visit their documentation](https://drand.love/docs/cryptography/#setup-phase) for more details on it). This implementation operates under the assumption of a p2p network, allowing operators to communicate.
+The `ssv-dkg` was built to lift this assumption and provide a communication layer that centered on an Initiator figure, to facilitate communication between operators. The introduced potential risk for centralization and bad actors is handled with signatures and signature verifications, as explained in the Security notes section.
+Finally, the outcome of the DKG ceremony is a BLS key pair to be used for validator duties by operators on the ssv.network. As such, the tool ends the process by creating a deposit file to activate the newly created validator key pair, and proceeds to generating the payload for the transaction, and triggering the transaction itself.
```

Comment:
we are not triggering the tx (last line).

Reply:
I had fixed it already, and in some copy-paste action, it made its way back! 🤦‍♂️